### PR TITLE
Add MSTest as supported test adapter

### PR DESCRIPTION
**Pull request by MarcoRossignoli**
@Evangelink @nohwnd wdyt?

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/testing/unit-testing-platform-intro.md](https://github.com/dotnet/docs/blob/310e9bdfdf1197c693a0077af8889d329e034222/docs/core/testing/unit-testing-platform-intro.md) | [Microsoft.Testing.Platform overview](https://review.learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-platform-intro?branch=pr-en-us-40350) |

### CODE REVIEW

File: docs/core/testing/unit-testing-platform-intro.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ Microsoft.Testing.Platform is a lightweight and portable alternative to [VSTest]
 
 `Microsoft.Testing.Platform` is open source. You can find `Microsoft.Testing.Platform` code in [microsoft/testfx](https://github.com/microsoft/testfx/tree/main/src/Platform/Microsoft.Testing.Platform) GitHub repository.
 
+# Supported test frameworks
+
+* MSTest a.k.a. [MSTest runner](unit-testing-mstest-runner-intro.md)
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-platform-intro.md:17]: I think the section is good. I would maybe changed the wording for MSTest runner.

Something like

MSTest. In MSTest, the support of Microsoft.Testing.Platform is done via MSTest runner.

(I'm on phone so I have kept the back quote or links)

Comment (MarcoRossignoli) [reply docs/core/testing/unit-testing-platform-intro.md]: applied

File: docs/core/testing/unit-testing-platform-intro.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ Microsoft.Testing.Platform is a lightweight and portable alternative to [VSTest]
 
 `Microsoft.Testing.Platform` is open source. You can find `Microsoft.Testing.Platform` code in [microsoft/testfx](https://github.com/microsoft/testfx/tree/main/src/Platform/Microsoft.Testing.Platform) GitHub repository.
 
+# Supported test frameworks
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-platform-intro.md:15]:
```suggestion
## Supported test frameworks
```

File: docs/core/testing/unit-testing-platform-intro.md [hunk 3]
```diff
@@ -12,6 +12,10 @@ Microsoft.Testing.Platform is a lightweight and portable alternative to [VSTest]
...
+
+* MSTest. In MSTest, the support of `Microsoft.Testing.Platform` is done via [MSTest runner](unit-testing-mstest-runner-intro.md).
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-platform-intro.md:17]:
```suggestion
* MSTest. In MSTest, the support of `Microsoft.Testing.Platform` is done via [MSTest runner](unit-testing-mstest-runner-intro.md).
* NUnit: work in progress - refer to https://github.com/nunit/nunit3-vs-adapter/issues/1152
* xUnit: work in progress - refer to https://github.com/xunit/visualstudio.xunit/issues/402
```

**Review by Evangelink — APPROVED** [review]

File: docs/core/testing/unit-testing-platform-intro.md [hunk 4]
```diff
@@ -12,6 +12,12 @@ Microsoft.Testing.Platform is a lightweight and portable alternative to [VSTest]
 
 `Microsoft.Testing.Platform` is open source. You can find `Microsoft.Testing.Platform` code in [microsoft/testfx](https://github.com/microsoft/testfx/tree/main/src/Platform/Microsoft.Testing.Platform) GitHub repository.
 
+## Supported test frameworks
+
+* MSTest. In MSTest, the support of `Microsoft.Testing.Platform` is done via [MSTest runner](unit-testing-mstest-runner-intro.md).
+* NUnit: work in progress - refer to https://github.com/nunit/nunit3-vs-adapter/issues/1152
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-platform-intro.md:18]:
```suggestion
* NUnit: work in progress, for more information, see [https://github.com/nunit/nunit3-vs-adapter/issues/1152](Microsoft Testing Platform for NUnit).
```

File: docs/core/testing/unit-testing-platform-intro.md [hunk 5]
```diff
@@ -12,6 +12,12 @@ Microsoft.Testing.Platform is a lightweight and portable alternative to [VSTest]
...
+* xUnit: work in progress - refer to https://github.com/xunit/visualstudio.xunit/issues/402
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-platform-intro.md:19]:
```suggestion
* xUnit: work in progress, for more information, see [Microsoft Testing Platform for xUnit](https://github.com/xunit/visualstudio.xunit/issues/402).
```

**Review by IEvangelist — APPROVED** [review]
Nits

File: docs/core/testing/unit-testing-platform-intro.md [hunk 6]
```diff
@@ -12,6 +12,12 @@ Microsoft.Testing.Platform is a lightweight and portable alternative to [VSTest]
 
 `Microsoft.Testing.Platform` is open source. You can find `Microsoft.Testing.Platform` code in [microsoft/testfx](https://github.com/microsoft/testfx/tree/main/src/Platform/Microsoft.Testing.Platform) GitHub repository.
 
+## Supported test frameworks
+
+* MSTest. In MSTest, the support of `Microsoft.Testing.Platform` is done via [MSTest runner](unit-testing-mstest-runner-intro.md).
+* NUnit: work in progress, for more information, see [https://github.com/nunit/nunit3-vs-adapter/issues/1152](Microsoft Testing Platform for NUnit).
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-platform-intro.md:18]:
```suggestion
* NUnit: work in progress, for more information, see [Microsoft Testing Platform for NUnit](https://github.com/nunit/nunit3-vs-adapter/issues/1152).
```

**Review by Evangelink — APPROVED** [review]

**Pull request merged**